services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -257,13 +300,17 @@
           folder                        (validate-publish-dest folder)
           folder-id                     (uuidify (:id folder))
           {:keys [irods-avus metadata]} (metadata-get user folder-id)
+          template-id                   (-> metadata :templates first :template_id)
           ezid-metadata                 (parse-valid-ezid-metadata irods-avus metadata)
           ezid-response                 (ezid/mint-id shoulder ezid-metadata)
           identifier                    (get ezid-response (keyword type))
-          publish-path                  (publish-data-item user folder (ezid-response->avus ezid-response))]
+          alt-identifiers               (dissoc ezid-response (keyword type))
+          publish-avus                  (format-publish-avus metadata ezid-metadata identifier alt-identifiers)
+          publish-path                  (publish-data-item user folder)]
       (email/send-permanent-id-request-complete type
                                                 publish-path
                                                 (json/encode ezid-response {:pretty true}))
+      (publish-metadata folder template-id publish-avus)
```

Comment:
Actually, this is intentional, because the email will contain the new identifier(s), so if something goes wrong when setting the metadata, the admin can manually set the metadata from the email.